glusterd2/options/options.go

```diff
@@ -347,11 +349,11 @@ func ValidateSizeList(o *Option, val string) error {
 			}
 			switch size {
 			case "KB", "kb":
-				sizeinbytes = v * 1024
+				sizeinbytes = v * deviceutils.KiB
```

Comment:
this should be `deviceutils.KiB` or `deviceutils.KB`

Reply:
same applies for other cases `iB` or `B`

Reply:
we have used Kib = 1024 and KB = 1000 so here we will have to use KiB instead of KB